app/graphql/types/wallets/recurring_transaction_rules/object.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Types
+  module Wallets
+    module RecurringTransactionRules
+      class Object < Types::BaseObject
+        graphql_name 'RecurringTransactionRule'
+
+        field :id, ID, null: false
+
+        field :rule_type, Types::Wallets::RecurringTransactionRules::RuleTypeEnum, null: false
+        field :interval, Types::Wallets::RecurringTransactionRules::IntervalEnum, null: true
+        field :threshold_credits, GraphQL::Types::Float, null: true
+        field :paid_credits, GraphQL::Types::Float, null: false
+        field :granted_credits, GraphQL::Types::Float, null: false
```

Comment:
GraphQL/OrderedFields: Fields should be sorted in an alphabetical order within their section. Field `granted_credits` should appear before `paid_credits`.